config/jobs/kubernetes-sigs/promo-tools/promo-tools.yaml

```diff
@@ -32,6 +32,13 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
+        resources:
+          limits:
+            cpu: 1
+            memory: 4Gi
+          requests:
+            cpu: 1
+            memory: 4Gi
```

Comment:
The CPU limit seems too low to me. This job runs in the default cluster, I'd just drop those resource requests and limits because they're not required in the default cluster and we also don't have a way to monitor if those are correct in that cluster.